src/app/content/studyGuides/components/UsingThisGuide/UsingThisGuideBanner.tsx

```diff
@@ -100,13 +124,15 @@ const UsingThisGuideBanner = (props: Props) => {
       <FormattedMessage id='i18n:studyguides:popup:using-this-guide:alt'>
         {(msg: Element | string) => <picture>
           <source media={theme.breakpoints.mobileMediumQuery} srcSet={mobileBanner} />
-          <DesktopBanner src={desktopBanner} alt={msg} tabIndex={0}/>
+          <DesktopBanner src={desktopBanner} alt={msg} tabIndex={0} ref={desktopBannerRef}/>
```

Comment:
It probably shouldn't be called `DesktopBanner` since it's displayed on all resolutions.